### PR TITLE
Italic the heading

### PR DESCRIPTION
**Pull request by JP109**
Changed header to italic style

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
-
+	font-style: italic;
 }
 
 #paragraph{
 	width:70%;
 	font-size:larger;
 	font-weight:lighter;
-	background:rgb(229, 227, 227);
+	background:#D1D1D1;
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:18]: Undo this change @JP109

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 1 above)
Comment (JP109) [reply assets/css/styles.css:18]: Sure

**Comment by JP109** [comment]
Done!

File: assets/css/styles.css (hunk 1 above)
Comment (JP109) [reply assets/css/styles.css]: Done!

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @JP109
